src/schemas/core/link.schema.ts

```diff
@@ -0,0 +1,11 @@
+import z from 'zod';
+
+export const urlSchema = z.string().url();
+
+export const urlsArraySchema = z.array(urlSchema);
+
+export const fileMetadataSchema = z.object({
+  fileName: z.string().min(1),
```

Comment:
is `fileName` included in the url?
if yes, is there a need to duplicate it's value